eZ/Publish/API/Repository/ContentService.php

```diff
@@ -305,12 +305,14 @@ public function deleteVersion(VersionInfo $versionInfo);
      * Loads all versions for the given content.
      *
      * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the user is not allowed to list versions
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException if the given status is invalid
      *
      * @param \eZ\Publish\API\Repository\Values\Content\ContentInfo $contentInfo
+     * @param null|int $status
```

Comment:
nitpick : int|null,
this is covered by new cs-fixer afair.

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2652/commits/25fb74fedbf28642bf71cb71002d61587af50313.